build.gradle

```diff
@@ -62,10 +60,6 @@ jar {
 javadoc {
   options.encoding = 'UTF-8'
   options.charSet = 'UTF-8'
-  options.links(
-          'http://docs.guava-libraries.googlecode.com/git-history/v18.0/javadoc/',
```

Comment:
Any explanation for the removal or even just touching this?

Reply:
I removed the guava dependency since it wasn't used in the project.  If you don't want me to do this I can put it back in.

Reply:
I guess it's more of "why are you touching the build script if the PR is focusing on cleaning performers?" But I get that it was already recognized that with the build script changes being a separate commit, that makes sense.